zul/src/archive/web/js/zul/inp/Slider.js

```diff
@@ -233,11 +233,19 @@ zul.inp.Slider = zk.$extends(zul.Widget, {
 			jq(document).unbind('zmouseup', widget.onup_);
 	},
 	doMouseDown_: function (evt) {
+		if (this._mold == 'knob') {

```

Comment:
As I said previously, you don't need to modify zul widget class because we will create a new class (extends zul.inp.Slider) to override what we want.

Reply:
As we discussion on 7/28, we keep "if (this._mold == 'knob')" to prevent the thread going through the this._mold != 'knob' for code effiecincy
.